### ISSUE TITLE
Selenium 3 is incompatible with Python 3.9

### ISSUE THREAD

**Issue by tari** [SeleniumHQ/selenium#8762]
## 🐛 Bug Report

There may be more issues than just this one, but in running tests on Python 3.9 I've encountered this error:

```
Traceback (most recent call last):
  File "/builds/cemetech/7/archives/tests/test_create_file.py", line 293, in testAutofillsFilename
    file_input.send_keys(tempfile1.name)
  File "/builds/cemetech/7/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 475, in send_keys
    value = self._upload(local_file)
  File "/builds/cemetech/7/.venv/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 695, in _upload
    content = base64.encodestring(fp.getvalue())
AttributeError: module 'base64' has no attribute 'encodestring'
```

This appears to have been fixed in #7446, but is not present in any stable releases on pypi.

Either the 3.x series on PyPi should be marked as incompatible with Python 3.9, or #7446 should be backported and a new Python release cut.

## Environment

OS: Linux
Browser: Firefox
Language Bindings version: Python 3.141.0

**Comment by AutomatedTester** [comment]
Hi

Thanks for the issue. We won't be releasing another version 3 as we're heading to finishing off Selenium 4. It is a drop in replacement to use Selenium 4.0.0.a5 so should work the same. There should not be any breaking changes.

**Comment by Surabhi228** [comment]
is this resolved? how to install selenium in python 3.9
